docs/python/examples/widget-store.md

```diff
@@ -254,7 +254,7 @@ def crash_application():
     os._exit(1)
 ```
 ## Try it Yourself!
-### Deploying to the Cloud
+### Deploying to DBOS Cloud
```

Comment:
Should we focus on deploying it locally first?

Reply:
We can change the CTAs across all the demo apps in a future PR